tests/xilinx/latches.ys

```diff
@@ -0,0 +1,16 @@
+read_verilog latches.v
+
+proc
+flatten
+equiv_opt -assert -run :prove -map +/xilinx/cells_sim.v synth_xilinx # equivalency check
+async2sync
+equiv_opt -assert -run prove: -map +/xilinx/cells_sim.v synth_xilinx # equivalency check
+design -load postopt # load the post-opt design (otherwise equiv_opt loads the pre-opt design)
+
+design -load preopt
```

Comment:
Why two `design -load` in a row? The first one is getting wholly overwritten with the second, no?

Reply:
Good spot! Removed.